packages/airnode-node/test/fixtures/config/node-settings.ts

```diff
@@ -1,4 +1,5 @@
 import { NodeSettings } from '../../../src/config/types';
+import { version as packageVersion } from '../../../package.json';
```

Comment:
Nit: We have a function for returning the [package version](https://github.com/api3dao/airnode/blob/master/packages/airnode-node/src/version.ts#L3). Maybe you can use it for `airnode-node` code. In case the implementation changes (even though I doubt it).